extension/src/common/logger.test.ts

```diff
@@ -1,5 +1,13 @@
 import { Logger } from './logger'
 
+jest.mock('console', () => {
+  const actualModule = jest.requireActual('console')
+  return {
+    __esModule: true,
+    ...actualModule
+  }
+})
```

Comment:
[F] Unfortunately we cannot pull out a helper from this boilerplate as the compiler behaves differently if `jest.mock` in not written like this in the code.